pkg/kubectl/cmd/get.go

```diff
@@ -203,3 +217,17 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		return printer.PrintObj(r.Object, out)
 	})
 }
+
+func filterNonRunningPodsSelector() (fields.Selector, error) {
+	// TODO: use correct fields selector for pod list
+	field := "status.phase!=Succeeded,status.phase!=Failed"
```

Comment:
When I use this field selector to filter out terminated pods, it returns me correct result, but with some error message. Any suggestions?
```console
$ kubectl get pods
W0804 10:34:12.294753    5027 request.go:308] field selector: v1 - pods - status.phase - Failed: need to check if this is versioned correctly.
W0804 10:34:12.295358    5027 request.go:308] field selector: v1 - pods - status.phase - Succeeded: need to check if this is versioned correctly.
NAME                                           READY     STATUS    RESTARTS   AGE
fluentd-cloud-logging-kubernetes-minion-102n   1/1       Running   0          26d
fluentd-cloud-logging-kubernetes-minion-rkd8   1/1       Running   0          26d
```
```console
$ kubectl get pods -a
NAME                                           READY     STATUS       RESTARTS   AGE
fluentd-cloud-logging-kubernetes-minion-102n   1/1       Running      0          26d
fluentd-cloud-logging-kubernetes-minion-rkd8   1/1       Running      0          26d
hello-world                                    0/1       ExitCode:0   0          3d
hello-world-2                                  0/1       ExitCode:0   0          3d
```